File: DL/UsuarioCajero.cs
using System;
using System.Collections.Generic;

namespace DL;

public partial class UsuarioCajero
{
    public int IdUsuarioCajero { get; set; }

    public int? NoCuenta { get; set; }

    public int? IdCajero { get; set; }

    public int? CantidadRetiro { get; set; }

    public int? CantidadDeposito { get; set; }

    public DateTime? Fecha { get; set; }

    public virtual Cajero? IdCajeroNavigation { get; set; }

    public virtual Usuario? NoCuentaNavigation { get; set; }
}

File: PL/angularapp/src/app/modelos/UsuarioCajero.ts
import { Cajero } from "./Cajero";
import { Usuario } from "./Usuario";

export interface UsuarioCajero {
  idUsuarioCajero?: number;
  usuario?: Usuario;
  cajero?: Cajero;
  cantidadRetiro?: number;
  cantidadDeposito?: number;
}
